fxmanifest.lua

```diff
@@ -8,7 +8,7 @@ shared_scripts {
     'config.lua',
     '@qb-core/shared/locale.lua',
     'locales/en.lua',
-    'locales/*.lua'
```

Comment:
Why delete this?

This removes support for other languages.

Reply:
Resolving since it was added back.